rack-protection/lib/rack/protection/referrer_policy.rb

```diff
@@ -6,14 +6,20 @@ module Rack
   module Protection
     ##
     # Prevented attack::   Secret leakage, third party tracking
-    # Supported browsers:: mixed support
-    # More infos::         https://www.w3.org/TR/referrer-policy/
-    #                      https://caniuse.com/#search=referrer-policy
+    # Supported browsers:: Most modern browsers from 2018 onwards.  See https://caniuse.com/#search=referrer-policy for
```

Comment:
I'm guessing the inclusion of this in most of these classes predates widespread use of caniuse.com, so I've been trying to sorta summarize and send a direct link. I dunno, maybe these should be removed entirely?